src/bot.js

```diff
@@ -23,7 +23,7 @@ const { spikeUID, simoneUID } = getConsts();
 
 bot.on("ready", async () => {
   // when loaded (ready event)
-  bot.user.setActivity(`${PREFIX}help | ${PREFIX}info`, { type: "LISTENING" });
+  bot.user.setActivity(`${PREFIX}help | ${PREFIX}info`, { type: "PLAYING" });
```

Comment:
Also taken care of in #56 . If this passes review before that, #56 can likely be closed.